Sanity/runtest.sh

```diff
@@ -1198,29 +1193,32 @@ rlJournalStart
             # 10 - return
             popd
             popd
-
-        rlPhaseEnd
-    }
+        else
+            echo "DAST test is not executed, helm missing!"
```

Comment:
`rlLog`, `rlLogInfo`, or `rlLogWarning` might be better than just `echo`. The same below.

Reply:
Outdated, just for debugging purposes.